src/examples/subsidy.ts

```diff
@@ -0,0 +1,104 @@
+  }
+
+  console.log('Connecting to the node...\n');
+
+  const api = await getClient(process.env.ACCOUNT_SEED);
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const identity = (await api.getSigningIdentity())!;
+  console.log(`Connected! Signing Identity ID: ${identity.did}`);
+
+  // get subsidizer account
+  const { account: subsidizer } = await identity.getPrimaryAccount();
+
+  // granting a subsidy to beneficary
+  const subsidyGrantQ = await api.accountManagement.subsidizeAccount({
+    beneficiary,
+    allowance: new BigNumber(1000),
+  });
+  const subsidyGrantResult = await subsidyGrantQ.run();
+
+  console.log(`Subsidy request has been created and will expire in ${subsidyGrantResult.expiry}`);
+
+  // beneficiary now has to accept the subsidy request
+  // it is possible to run accept method on `subsidyGrantResult` however, it is more realistic to store the `authId` and then ask the beneficiary to accept the subsidy
+  // here as example we use BOB_SEED as the account that would accept the subsidy, in real life this would be signed by the actual beneficiary
+  const beneficiaryApi = await getClient(process.env.BOB_SEED);
+
+  const beneficiaryIdentity = (await beneficiaryApi.getSigningIdentity()) as Identity;
+
+  const {
+    account: beneficiaryAccount,
+  } = (await beneficiaryIdentity.getPrimaryAccount()) as PermissionedAccount;
+
+  const authorization = await identity.authorizations.getOne({ id: subsidyGrantResult.authId });
+  const acceptTx = await authorization.accept({ signingAccount: beneficiaryAccount });
+
+  console.log(
+    `Subsidy request has been accepted by the beneficiary. Transaction status: ${acceptTx.status}`
+  );
+
+  // get subsidy
+  const subsidy = api.accountManagement.getSubsidy({ subsidizer, beneficiary });
+
+  // get allowance
+  let allowance = await subsidy.getAllowance();
+  console.log(`Allowance before increase is: ${allowance}`);
+
+  // add allowance
+  const addAllowanceQ = await subsidy.increaseAllowance({ allowance: new BigNumber(1000) });
+  await addAllowanceQ.run();
+
+  allowance = await subsidy.getAllowance();
+  console.log(`Allowance after increase is: ${allowance}`);
+
+  // remove allowance
+  const removeAllowanceQ = await subsidy.decreaseAllowance({ allowance: new BigNumber(1000) });
+  await removeAllowanceQ.run();
+
+  allowance = await subsidy.getAllowance();
+  console.log(`Allowance after decrease is: ${allowance}`);
+
+  // set allowance
+  const setAllowanceQ = await subsidy.setAllowance({ allowance: new BigNumber(5000) });
+  await setAllowanceQ.run();
+
+  allowance = await subsidy.getAllowance();
+  console.log(`Allowance after setting is: ${allowance}`);
+
+  // quit subsidy
+  const quitSubsidyQ = await subsidy.quit();
+  await quitSubsidyQ.run();
+
+  // check if subsidy is still active
+  const subsidyExists = await subsidy.exists();
+  console.log(`Subsidy still exists: ${subsidyExists}`);
```

Comment:
```suggestion
  console.log(`Subsidy exists: ${subsidyExists}`);
```
The word still makes me think it should be "true" for some reason.